sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -119,13 +122,26 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
         );
     }
 
+    if (
+      educationProgram.sabcCode &&
+      (await this.hasExistingProgramWithSameSABCCode(
```

Comment:
for ease of readability can we assign `(await this.hasExistingProgramWithSameSABCCode(
        institutionId,
        educationProgram.sabcCode,
        programId,
      )` to a const and use the const in the expression? just a suggestion

Reply:
maybe we can move this before like 116

Reply:
yes, ok. Done. Thanks!